tests/integration/single_table/test_base.py

```diff
@@ -728,3 +733,26 @@ def test_get_info():
         'last_fit_date': today,
         'fitted_sdv_version': version
     }
+
+
+def test_synthesizer_with_inequality_constraint():
+    """Ensure that the ``Inequality`` constraint can sample from the model."""
+    # Setup
+    real_data, metadata = download_demo(
+        modality='single_table',
+        dataset_name='fake_hotel_guests'
+    )
+    synthesizer = GaussianCopulaSynthesizer(metadata)
+    checkin_lessthan_checkout = {
+        'constraint_class': 'Inequality',
+        'constraint_parameters': {
+            'low_column_name': 'checkin_date',
+            'high_column_name': 'checkout_date'
+        }
+    }
+
+    synthesizer.add_constraints([checkin_lessthan_checkout])
+    synthesizer.fit(real_data)
+
+    # Run and Assert
+    synthesizer.sample(num_rows=500)
```

Comment:
can we assert the constraint is valid in the sampled data? Do like synthesizer.validate(sampled_data) or something like that